src/Menu/Menu.js

```diff
@@ -72,8 +66,14 @@ export default function Menu({
   );
 }
 
+function isGroupedData(data) {
```

Comment:
Should this be exported and then referenced in Dropdown & Select (others)?

Reply:
Good suggestion.  I ended up moving `getItems` into Menu and importing it for use in Select/Dropdown.  This also meant that `isGroupedData` is no longer needed outside of Menu.